dash.js

```diff
@@ -779,17 +650,17 @@ const myDash = new Lang.Class({
                    !actor.animatingOut;
         });
 
-        let appIcons = iconChildren.map(function(actor){
+        let appIcons = iconChildren.map(function(actor) {
             return actor.child._delegate;
         });
 
       return appIcons;
     },
 
-    _updateAppIconsGeometry: function() {
+    _updateAppIcons: function() {
         let appIcons = this._getAppIcons();
-        appIcons.forEach(function(icon){
-            icon.updateIconGeometry();
+        appIcons.forEach(function(icon) {
+            icon.onWindowsChanged();
         });
```

Comment:
* _updateAppIconsGeometry was the name in the windows-lits extension, because it chenge the icon_geometry property of the metawindow. Why changing the name?

* Is there a reason for calling onWindowsChange? I prefer not to call something which is clearly meant to be a signal callback from here.